pandas_dev_flaker/_ast_helpers.py

```diff
@@ -30,3 +31,19 @@ def check_for_wrong_alias(
             return name_.asname != alias
     else:
         return False
+
+
+def is_str_constant(
+    node: ast.Call,
+) -> bool:
+    python_version = float(sys.version_info.major) + 0.1 * float(
+        sys.version_info.minor,
+    )
+    return isinstance(node.func, ast.Attribute) and (
+        (python_version < 3.8 and isinstance(node.func.value, ast.Str))
+        or (
+            python_version >= 3.8
+            and isinstance(node.func.value, ast.Constant)
+            and isinstance(node.func.value.value, str)
+        )
+    )
```

Comment:
Does it not work to just use
```
isinstance(node.func.value, ast.Str)
```
in all versions of Python?

Reply:
`ast.Str` is deprecated since python 3.8. It will be removed in future python releases, according to [ast](https://docs.python.org/3/library/ast.html).

Reply:
ah, I wasn't aware, thanks!